ios/FluentUI/Table View/ListVnext.swift

```diff
@@ -0,0 +1,225 @@
+
+    public init(cells: [MSFListVnextCell],
+                layoutType: MSFListCellVnextLayoutType,
+                iconStyle: MSFListIconVnextStyle) {
+        self.cells = cells
+        self.state = MSFListVnextState()
+        self.tokens = MSFListTokens(layoutType: layoutType, iconStyle: iconStyle)
+
+    }
+
+    public var body: some View {
+        List {
+            ForEach(cells, id: \.self) { item in
+                MSFListCellView(cell: item, tokens: tokens)
+                    .onTapGesture(perform: item.handler ?? {})
+            }
+            .border(state.hasBorder ? Color(tokens.borderColor) : Color.clear, width: state.hasBorder ? tokens.borderSize : 0)
+            .frame(minHeight: tokens.layoutType.height)
+            .listRowInsets(EdgeInsets())
+        }
+        .frame(width: UIScreen.main.bounds.width, height: 300)
+        .environment(\.defaultMinListRowHeight, 0)
+    }
+}
+
+extension MSFListView {
+    /// View for List Cells
+    struct MSFListCellView: View {
+        var cell: MSFListVnextCell
+        var tokens: MSFListTokens
+
+        init(cell: MSFListVnextCell, tokens: MSFListTokens) {
+            self.cell = cell
+            self.tokens = tokens
+        }
+
+        var body: some View {
+            HStack(spacing: 0) {
+                if let leadingView = cell.leadingView {
+                    Image(uiImage: leadingView)
+                        .resizable()
+                        .frame(width: tokens.iconSize, height: tokens.iconSize)
+                        .padding(.trailing, tokens.iconInterspace)
+                }
+                VStack(alignment: .leading) {
+                    if let title = cell.title {
+                        Text(title)
+                            .font(Font(tokens.textFont))
+                            .foregroundColor(Color(tokens.leadingTextColor))
+                            .lineLimit(cell.titleLineLimit)
+                    }
+                    if let subtitle = cell.subtitle {
+                        if subtitle != "" {
+                            Text(subtitle)
+                                .font(Font(tokens.subtitleFont))
+                                .foregroundColor(Color(tokens.subtitleColor))
+                                .lineLimit(cell.titleLineLimit)
+                        }
+                    }
+                }
+                Spacer()
+                ZStack(alignment: .trailing) {
+                    if let trailingView = cell.trailingView {
+                        if trailingView != .none {
+                            Image(uiImage: trailingView.icon!)
+                                .resizable()
+                                .foregroundColor(Color(trailingView == .disclosureIndicator ? tokens.disclosureIconForegroundColor : tokens.trailingItemForegroundColor))
+                                .frame(width: trailingView == .disclosureIndicator ? tokens.disclosureSize : tokens.iconSize,
```

Comment:
It's good practice to avoid repeating the property calls.
You can do this on the top of the method for the disclosureIndicator for example:

`let disclosureIndicatorSize = tokens.disclosureSize`

and use it throughout the implementation here.